src/higlass_schema/schema.py

```diff
@@ -549,25 +513,25 @@ def schema_extra(schema: Dict[str, Any], _) -> None:
     compactLayout: Optional[bool] = None
     exportViewUrl: Optional[str] = None
     trackSourceServers: Optional[List[str]] = None
-    views: Optional[Annotated[List[ViewT], Field(min_items=1)]] = None
+    views: Optional[Annotated[List[ViewT], Field(min_length=1)]] = None
     zoomLocks: Optional[ZoomLocks] = None
     locationLocks: Optional[LocationLocks] = None
     valueScaleLocks: Optional[ValueScaleLocks] = None
     chromInfoPath: Optional[str] = None
 
 
 def schema():
-    root = Viewconf.schema()
+    root = Viewconf.model_json_schema()
 
     # remove titles in defintions
-    for d in root["definitions"].values():
+    for d in root["$defs"].values():
         d.pop("title", None)
 
     # nice ordering, insert additional metadata
     ordered_root = OrderedDict(
         [
-            ("$schema", "http://json-schema.org/draft-07/schema#"),
-            ("$id", "https://higlass.io/#viewconf"),
+            ("$schema", GenerateJsonSchema.schema_dialect),
+            ("$id", "https://higlass.io/schemas/viewconf"),
```

Comment:
The "$id" field in JSON Schema is used to define a unique identifier (URI) for the schema. I copied the previous one from the [original schema](https://github.com/higlass/higlass/blob/develop/app/schema.json), but apparently it's not a real link. 

Is the URL you specified supposed to be a real link to the schema? Otherwise I say we remove it.

Reply:
Mine was made up too.